modules.py

```diff
@@ -268,7 +285,6 @@ def resize_psf(self, wavelength=.76, size=505, scale=0.110):
         print "resizing PSF to match pixel resolution of %s''/px..." % scale
```

Comment:
You are using the 'print' function to report logging messages.  Have a look at the 'logging' module and try using that instead.  Then you can have verbosity levels.